realm/realm-library/src/main/java/io/realm/internal/Group.java

```diff
@@ -226,6 +226,15 @@ public void writeToFile(File file, byte[] key) throws IOException {
         return nativeWriteToMem(nativePtr);
     }
 
+    /*
+     * Check if the Group contains any objects.
```

Comment:
Also add that this only checked for `class_` tables, e.g. this would return true even if the `pk` table contained information.